GoogleSignIn/Sources/Public/GoogleSignIn/GIDGoogleUser.h

```diff
@@ -53,6 +61,12 @@ NS_ASSUME_NONNULL_BEGIN
 /// see https://developers.google.com/identity/sign-in/ios/backend-auth.
 @property(nonatomic, readonly, nullable) GIDToken *idToken;
 
+/// A new authorizer for `GTLService`, `GTMSessionFetcher`, or `GTMHTTPFetcher`.
```

Comment:
We should be returning the same authorizer instance every time the property is accessed, rather the the old behavior which creates "A new authorizer" every time the method is called.

Reply:
Made the change in this PR.